### DISCUSSION TITLE
PK-Sim FcRn recycling fraction to interstitial space

### DISCUSSION THREAD

**Discussion by akim2000** [Open-Systems-Pharmacology/Forum#1745] [Questions & Problems]
As I was working on building a large molecule brain model in MoBi, I noticed that PK-Sim has the Fraction recycled to plasma (or commonly referred to as FR in literature) set to 1, which suggests that effectively no recycling of drug–FcRn or endogenous IgG–FcRn from the endosome to interstitial space occurs. Additionally, it also has Fraction of endosomal uptake from plasma set to 1, which does not allow for any transport from interstitial space back to the endosome since the passive transport formula includes (1-Fraction of endosomal uptake). 

The explanation given in Niederalt (2017) says "In order to prevent the net extravasation via the endosomal space and to restrict extravasation to the two-pore mechanism, f up vas and f rec vas were both set to 1 for all simulations in all species, i.e., no uptake from interstitial space to the endosomal space and no recycling to interstitial space was taken into account."

However, I feel like this does not really justify neglecting FcRn-mediated transcytosis into the interstitial space, as literature does suggest bidirectional transport via FcRn-mediated transcytosis from apical to basal and basal to apical transport (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4451002/). I know that the value of FR = 0.715 that is commonly used in some of the large molecule PBPK models was derived from fitting the parameter to observed data and not a physiological value obtained from experiments, but especially in organs such as the brain, where the two-pore model (convective transport and diffusion) is almost negligible due to the blood-brain barrier (maybe a bit more relevant for BCSFB), I think transcytosis can play a significant role in determining tissue concentrations. 

Is there another specific reason other than the one stated above for PK-Sim to not include transcytosis via FcRn? Why does PK-Sim focus on restricting extravasation only to the two-pore mechanism?

**Comment by wilbertdew** [comment]
Hi @akim2000,

The main reason indeed seems to be the separation of clearance and distribution mechanisms and the benefit this brings for parameter identifiability. For antibodies or antibody fragments, the difference between the two parameterizations seems to be small as well for most tissues. Exceptions would be the gut, the placenta, and maybe the brain, so it would make sense to explore different versions of these fractions in my opinion.
